From be5247b8dcb412a7172c1261151f1af812831d79 Mon Sep 17 00:00:00 2001
From: Joshua Strobl <joshua@streambits.io>
Date: Fri, 20 Sep 2019 12:59:26 +0300
Subject: [PATCH 1/1] Drop -Werror from conf cflags

In GLib 2.62+, G_TYPE_INSTANCE_GET_PRIVATE is deprecated and affects macros like BAMF_APPLICATION_GET_PRIVATE.
---
 configure.ac | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/configure.ac b/configure.ac
index 41cb7db..7cc3b9d 100644
--- a/configure.ac
+++ b/configure.ac
@@ -155,7 +155,7 @@ AC_SUBST(COVERAGE_LDFLAGS)
 AM_PATH_GTK_3_0
 
 dnl CFLAGS
-CFLAGS="$CFLAGS -Wall -Werror -Wno-error=deprecated-declarations -Wempty-body -Wformat-security -Winit-self -Warray-bounds -Wimplicit-function-declaration -lm"
+CFLAGS="$CFLAGS -Wall -Wno-error=deprecated-declarations -Wempty-body -Wformat-security -Winit-self -Warray-bounds -Wimplicit-function-declaration -lm"
 
 AC_SUBST(BAMFDAEMON_CFLAGS)
 AC_SUBST(BAMFDAEMON_LIBS)
-- 
2.23.0

